packages/portal-proto/src/features/layout/Header.tsx

```diff
@@ -151,7 +151,7 @@ export const Header: React.FC<HeaderProps> = ({
           </a>
           <Link href="/manage_sets" passHref>
             <a
-              data-testid="link-header-manage-sets"
+              data-testid="button-header-manage-sets"
```

Comment:
I know that you have it as such in the automation but is it okay to change it there to make it consistent as this is now a link?

Reply:
The items on the headers need to be all buttons or all links, they cannot be both